docs/UserGuide.md

```diff
@@ -247,16 +314,16 @@ Here are some recommended features to explore after this tutorial.
 
 **:information_source: Notes about the command format:**<br>
 
-* Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+* Words in `UPPER_CASE` are the parameters to be supplied by you.<br>
+  e.g. `addc n/NAME ...`, `NAME` is a parameter which can be used as `addc n/John Doe ...`.
 
 * Items in square brackets are optional.<br>
-  e.g `n/NAME [p/PHONE_NUMBER]` can be used as `n/John Doe p/12341234` or as `n/John Doe`.
+  e.g. `...n/NAME [q/QUANTITY]` can be used as `...n/Cake q/2` or as `...n/Cake`.
 
 * Items in curly brackets are an indication to use one of the given options, separated by the character `|`.
   e.g. `ct/{ind|ent}` is a parameter that should be used as `ct/ind` or `ct/ent` exactly.
 
-* Parameters can be in any order.<br>
+* Parameters can be in any order by you.<br>
```

Comment:
This also sounds a little off, maybe you can say "You can enter parameters in any order" instead

Reply:
wait idk why i added by you just like that HAHAHAH thanks for the catch